### PR TITLE
Feature - Reintroduce the installation command for the admin panel

### LINKED ISSUE #1417: Reintroduce the installation command for the admin panel
Currently, there is no option to create a Staff login when installing `v1.0`.

The install command from the previous hub needs to be implemented. See https://github.com/lunarphp/lunar/blob/0.7/packages/admin/src/Console/Commands/InstallHub.php

We don't have any custom assets to be publishing this time, but we might want to incorporate publishing Filament's assets perhaps?

### PR DESCRIPTION
**Pull request by lguichard**
Closes #1417

### CODE REVIEW

File: packages/core/src/Console/InstallLunar.php [hunk 1]
```diff
@@ -237,12 +293,35 @@ public function handle(): void
...
+            $lastname = $this->ask('Whats your last name?');
+            $email = $this->ask('Whats your email address?');
+            $password = $this->secret('Enter a password');
+
+            /** @var Staff $staff */
+            $staff = Staff::create([
+                'firstname' => $firstname,
+                'lastname' => $lastname,
+                'email' => $email,
+                'password' => bcrypt($password),
```
Comment (wychoong) [inline packages/core/src/Console/InstallLunar.php]: should be 
```php
use Illuminate\Support\Facades\Hash;

Hash::make($password)
```

File: packages/core/src/Console/InstallLunar.php [hunk 2]
```diff
@@ -237,12 +293,35 @@ public function handle(): void
...
+                'admin' => true,
+            ]);
+
+            $staff->syncRoles('admin');
         }
 
+        
+        $this->comment('Publishing assets...');
+        $this->publishAssets();
```
Comment (wychoong) [inline packages/core/src/Console/InstallLunar.php:319]: filament's recommendation is to add this to composer script, is adding this here fix or any reason to have this run after lunar?

File: packages/core/src/Console/InstallLunar.php [hunk 3]
```diff
@@ -237,12 +293,35 @@ public function handle(): void
...
+        $this->newLine();
+
+        $this->comment('Cache clear...');
+        $this->cleaning();
```
Comment (wychoong) [inline packages/core/src/Console/InstallLunar.php]: any reason why this need to be run after install?

**Comment by glennjacobs** [comment]
You have worked on an issue assigned to me. There's quite a lot I'd do differently, so I'll take over this PR.

**Comment by lguichard** [comment]
> You have worked on an issue assigned to me. There's quite a lot I'd do differently, so I'll take over this PR.

Ok i didn’t see the issue assigned.

**Comment by lguichard** [comment]
Closed by another PR.

**Pull request closed without merging**